README.md

```diff
@@ -28,6 +28,9 @@ var config = {
 }
 ```
 
+You might need to specify the country in the ``stationName`` if your station is not found. In our examply, it would be
+``Bangkok University Rangsit Campus, Pathum Thani, India``
```

Comment:
I'm pretty sure Bangkok is in Thailand ;-)
Could you change it and I will merge the PR

Thanks!

Reply:
I'm so sorry 🤦‍♂️😅. Made the pull request in a total rush this morning because I added the publication of the PR to my to-do list when I first started playing with the repository at the beginning of the week and I totally forgot about it, until this morning. Rush is not a good idea, never :)

Thank you very much for pointing it out. Give me a sec and it will be done.

Reply:
Thanks! 😄